src/validators.js

```diff
@@ -37,7 +39,9 @@ function isJSONString(body) {
 
 function isJSONObject(body) {
   try {
-    assert.strictEqual(typeof body, 'object');
+    if (typeof body !== 'object' || Array.isArray(body) || body === null) {
```

Comment:
This is technically doing more conditional checks than the original. Just calling that out in case there's anything there. Looks like you added some tests for this though.

Reply:
Yeah it was added to catch an edge case. So I guess in a sense this is technically a bug fix?